dynamicio/io/postgres.py

```diff
@@ -107,6 +111,7 @@ def check_injections(self) -> None:
 
     def read(self) -> pd.DataFrame:
         """Handles Read operations for Postgres."""
+        self.check_injections()
         if not bool(self.sql_query) ^ bool(self.table_name):  # Xor
```

Comment:
Shall we put brackets like this 

`if not (bool(self.sql_query) ^ bool(self.table_name)):`

It's probably well defined, but at first glance it's confusing whether the not operates on the whole expression or the first boolean